src/_tutorials/dart-vm/cmdline.md

```diff
@@ -104,62 +106,64 @@ This tutorial goes into detail about this app in the following sections.
 For a brief look now, hover over the highlighted code below for explanations.
 
 <pre class="prettyprint lang-dart">
-import 'dart:async';
-import 'dart:convert';
 import 'dart:io';
+import 'dart:convert';
+<a href="#" class="dart-popover" data-toggle="popover" title="Asynchronous library" data-html="true" data-trigger="hover focus" data-content="The dart:async library defines Future and Stream classes">import 'dart:async';</a>
 
 import 'package:args/args.dart';
 
-const LINE_NUMBER = 'line-number';
-var NEWLINE = '\n';
+const lineNumber = 'line-number';
 
 ArgResults argResults;
 
 void main(<a href="#" class="dart-popover" data-toggle="popover" title="Command-line arguments" data-html="true" data-trigger="hover focus" data-content="Command-line arguments are passed in by the system when the program starts.">List&lt;String&gt; arguments</a>) {
+  exitCode = 0; //presume success
   final parser = new ArgParser()
-      ..addFlag(LINE_NUMBER, negatable: false, abbr: 'n');
+      ..addFlag(lineNumber, negatable: false, abbr: 'n');
 
   <a href="#" class="dart-popover" data-toggle="popover" title="Arguments parser" data-html="true" data-trigger="hover focus" data-content="The ArgParser class provides help with parsing command-line arguments.">argResults = parser.parse(arguments);</a>
   List&lt;String&gt; paths = argResults.rest;
 
-  dcat(paths, argResults[LINE_NUMBER]);
+  dcat(paths, argResults[lineNumber]);
 }
 
-Future dcat(List&lt;String&gt; paths, bool showLineNumbers) {
+<a href="#" class="dart-popover" data-toggle="popover" title="Asynchronous function" data-html="true" data-trigger="hover focus" data-content="An asynchronous function is marked with 'async' and returns a Future.">Future</a> dcat(List&lt;String&gt; paths, bool showLineNumbers) <a href="#" class="dart-popover" data-toggle="popover" title="Asynchronous function" data-html="true" data-trigger="hover focus" data-content="An asynchronous function is marked with 'async' and returns a Future.">async</a> {
```

Comment:
It's strange to have single quotes ('async'). Could you change the surrounding quotes to ' and use " instead for "async"? (This happens elsewhere too, so if it's too big a change, fuhgeddabowdit.)

It's also strange to have the same highlight text twice. Maybe just have it on the async part, not Future?